README.md

```diff
@@ -17,39 +17,38 @@ Daffodil is a frontend toolkit that intends to accomplish three things:
 
 * :free: Forever Free, Open Source, and MIT Licensed
 * :hammer: Platform Agnostic Drivers For Your Platform of Choice
-    * Shopify
-    * Magento 2
-    * In-Memory Backend
-        * Prebuilt
+    * Shopify (WIP, 5% coverage)
+    * Magento 2 (WIP, 50% coverage)
+    * In-Memory Backend (60% coverage)
+        * Prebuilt, in-browser, randomly generated
         * :zap: Great for lightning fast iteration and concepting.
-* :cake: Decoupled and Composable Packages for your use case
-    * Product
-    * Cart
-    * Checkout
-* :books: Well Documented
-    * Example Implementation
-    * API References
-    * Tutorials
+* :cake: [Decoupled and Composable Packages for your use case](#packages)
+* :books: Well Documented ([WIP: We need **your** help!](https://github.com/graycoreio/daffodil/issues/336))
 * :bulb: Component Kit
-    * Purely Functional
+    * Functionally Written (Lambda-like)
```

Comment:
What does `Lambda-like` mean?

Reply:
I'll add a doc describing this in another PR and revert this change.

Reply:
I've updated this.